clang/test/Sema/uninit-variables.c

```diff
@@ -551,3 +551,14 @@ struct full_of_empty empty_test_2(void) {
   struct full_of_empty e;
   return e; // no-warning
 }
+
+struct with_explicit_field {
+  int x;
+  int y [[clang::requires_explicit_initialization]]; // expected-note {{declared}}
+};
+
+void aggregate() {
+  struct with_explicit_field a; // expected-warning {{is not explicitly initialized}}
```

Comment:
For these diagnostics, please ensure you do the whole diagnostic instead of partial match.